app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomPresenter.kt

```diff
@@ -16,7 +16,18 @@ import chat.rocket.android.core.lifecycle.CancelStrategy
 import chat.rocket.android.helper.MessageHelper
 import chat.rocket.android.helper.UserHelper
 import chat.rocket.android.infrastructure.LocalRepository
-import chat.rocket.android.server.domain.*
+import chat.rocket.android.server.domain.ChatRoomsInteractor
```

Comment:
Did you remove the wildcard and start adding the imports manually? Just a curiosity.
Thanks for doing that @leonardoaramaki 👊

Reply:
No, actually I configured AS code style settings by increasing the maximum number of imports to a very high value before replacing it to a wildcard. :+1: